src/konserve_jdbc/core.clj

```diff
@@ -197,7 +197,17 @@
     (if (:sync? env) nil (go-try- nil)))
   (-create-store [_ env]
     (async+sync (:sync? env) *default-sync-translation*
-                (go-try- (jdbc/execute! connection (create-statement (:dbtype db-spec) table)))))
+                (go-try- 
+                  ;; Using CREATE IF NOT EXISTS is regarded as a schema change. To allow the store to be used
+                  ;; where schema changes are not allowed on production e.g. planetscale or the user does have schema permissions,
+                  ;; we test for existence first. This triggers an exception if it doesn't exist which we catch. 
+                  ;; Testing for existence in other ways is not worth the effort as it is specific to the db setup 
+                  ;; not just the type
+                  (let [res (try 
+                              (jdbc/execute! connection [(str "select 1 from " table " limit 1")])
+                              (catch Exception _e (debug (str "Table " table " does not exist. Attempting to create it.")) nil))]
```

Comment:
I think, at least `nil` should be on a line on its own so it is easier to see that `res` is assigned `nil` in case of an error. 
I would probably also put the `debug` statement on a newline so the catch body statements are separated from the rest of the parameters.

Reply:
@jsmassa good point. Made the changes